src/Project/ToolRunner.ts

```diff
@@ -83,13 +91,10 @@ export class ToolRunner {
     return oneccRealPath;
   }
 
-  public getRunner(name: string, onecc: string, tool: string, toolargs: ToolArgs, path: string) {
-    return new Promise<string>((resolve, reject) => {
-      this.logger.outputWithTime('Running: ' + name);
-      // place tool to the front of toolargs
-      toolargs.unshift(tool);
-      let cmd = cp.spawn(onecc, toolargs, {cwd: path});
-      this.handlePromise(resolve, reject, cmd);
+  public getRunner(job: Job, path: string) {
```

Comment:
You need to update a `ToolRunner.test.ts` file. This changes made a build break.

Reply:
Yes, I didn't care about the tests 😂